src/debug/PythonDebugProvider.ts

```diff
@@ -39,7 +39,9 @@ async function getPythonCommand(host: string, port: number): Promise<string> {
 
         // tslint:disable-next-line:strict-boolean-expressions
         if (pyExtension.exports && pyExtension.exports.debug) {
-            return (await pyExtension.exports.debug.getRemoteLauncherCommand(host, port)).join(' ');
+            const debugPyCommand: string = (await pyExtension.exports.debug.getRemoteLauncherCommand(host, port, false)).join(' ');
+            const debugPyFlag: string = '--configure-subProcess False';
```

Comment:
The point of the Python extension's `getRemoteLauncherCommand` method is to abstract away the debug tool. In fact, they switched from "ptvsd" to "debugpy" under the covers a while ago and our extension didn't have to do anything at all. Since "--configure-subProcess" is specific to "debugpy", I think we would need to add that as a parameter to `getRemoteLauncherCommand` similar to `waitUntilDebuggerAttaches` in their API

Reply:
Hi @ejizba, actually, just tested and I don't even need to disable the subprocess to make this work. I amended the PR and please check this again. Thanks